src/components/Question.jsx

```diff
@@ -119,25 +119,46 @@ export default class Question extends React.Component {
     this.setState({ showIcon: false });
   };
 
-  render() {
+  _getHeaderClassName = () => {
     const question = this.props.question;
-    const subQuestion = question[Constants.HAS_SUBQUESTION];
+    const collapsible = this.props.collapsible;
     const options = this.context.options;
-    const questionComponent = this.renderQuestion(question);
-    if (FormUtils.isHidden(question)) {
-      return null;
-    }
+    const subQuestion = question[Constants.HAS_SUBQUESTION];
+
+    let headerClassName = [];
+
+    headerClassName.push(
+      Question.getEmphasizedOnRelevantClass(question),
+      "question-header"
+    );
+
+    if (FormUtils.isWizardStep(question))
+      headerClassName.push("wizard-step bg-primary text-white");
+    if (FormUtils.isEmphasised(question))
+      headerClassName.push(Question.getEmphasizedClass(question));
+    if (FormUtils.isSection(question))
+      headerClassName.push("section-background");
+    if (collapsible) headerClassName.push("cursor-pointer");
     if (
-      !FormUtils.isRelevant(question) &&
-      (this.context.options.debugMode ||
+      FormUtils.isRelevant(question) &&
+      (options.debugMode ||
         JsonLdObjectUtils.checkId(subQuestion, options.startingQuestionId))
```

Comment:
Looking at code:
```
  static checkId(jsonLdObject, id) {
    if (jsonLdObject === undefined) {
      return;
    }
    if (jsonLdObject.constructor === Array) {
      return !!jsonLdObject.find((o) => o["@id"] === id);
    }
    if (jsonLdObject.constructor === Object) {
      return jsonLdObject["@id"] === id;
    }
  }
```

Is one level check sufficient? What if i have form:
- disabled-q-1
  - disabled-q-2
    - disabled-q-3
      - question-with starting-question-id 
      ?

Reply:
You can put it into a separate ticket if you want.

Reply:
Ok, covered in #172